airflow/models/connection.py

```diff
@@ -119,6 +120,8 @@ class Connection(Base, LoggingMixin):
     is_extra_encrypted = Column(Boolean, unique=False, default=False)
     _extra = Column('extra', String(5000))
 
+    UniqueConstraint(conn_id, name='unique_conn_id')
```

Comment:
This looks odd -- I would have expected it to be this
```suggestion
    __table_args = (
        UniqueConstraint(conn_id, name='unique_conn_id'),
    )
```

Reply:
Actually - even simpler is adding `unique=True` on Line 111: `conn_type = Column(String(500))`- ? `conn_type = Column(String(500), unique=True)`